app/src/test/java/com/aws/amazonlocation/data/response/ResponseDataTest.kt

```diff
@@ -42,26 +41,10 @@ class ResponseDataTest {
     @Test
     fun navigationDataTest() {
         val navigationData = Responses.RESPONSE_NAVIGATION_DATA_CAR_STEP_1
-        navigationData.getRegions()
         navigationData.destinationAddress = TEST_DATA
-        navigationData.getRegions()
         navigationData.subRegion = ""
-        navigationData.getRegions()
         navigationData.region = ""
-        navigationData.getRegions()
         navigationData.country = ""
-        navigationData.getRegions()
-        navigationData.region
-        navigationData.country
-        navigationData.duration
-        navigationData.distance
-        navigationData.startLat
-        navigationData.startLng
-        navigationData.endLat
-        navigationData.endLng
-        navigationData.destinationAddress
-        navigationData.subRegion
-        navigationData.country
         Assert.assertTrue(TEST_FAILED_NAVIGATION_DATA, navigationData.country == "")
```

Comment:
With the calls to getRegions() removed, what is this test still testing?

Reply:
Fixed, added comment